src/erlang-lib/ec_genet/src/ec_genet_server.erl

```diff
@@ -262,8 +261,7 @@ set_case(Tctx, HLPath, Choice, Case) ->
 %%% Mapping Function Interface
 %%%===================================================================
 
-throw_callmap_error(CallmapError,CallmapReason,Name) ->
-    Trace = erlang:get_stacktrace(),
+throw_callmap_error(CallmapError,CallmapReason,CallmapTrace,Name) ->
     FileLine =
         case Trace of
```

Comment:
I think this cannot compile, the variable is called `CallmapTrace` in the function head.